bindings/gympp.i

```diff
@@ -5,15 +5,19 @@
 #include "gympp/Common.h"
 #include "gympp/Environment.h"
 #include "gympp/gazebo/IgnitionEnvironment.h"
+#include "gympp/gazebo/GazeboWrapper.h"
 #include "gympp/gazebo/RobotSingleton.h"
 #include "gympp/GymFactory.h"
 #include "gympp/Metadata.h"
 #include "gympp/Robot.h"
 #include "gympp/Space.h"
+#include <stdint.h>
```

Comment:
Nitpick: for a C++ file, the recommended include is `cstdint`(see https://stackoverflow.com/questions/13642827/cstdint-vs-stdint-h). Or perhaps you did this for some weird SWIG incompatibility.

Reply:
I read https://github.com/swig/swig/issues/495 and I forgot to use the C++ styled inclusion. I did a test and using `cstdint` in the include list works, but `stdint.h` should be passed to the swig's `%include` directive in order to find the file.